compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenDialect.cpp

```diff
@@ -63,6 +64,22 @@ IREECodegenDialect::verifyOperationAttribute(Operation *op,
   //         b. It must have exactly one argument type, and the argument must be
   //         of type `transform::AnyOpType`.
 
+  if (symbol == kTuningSpecDefaultEntrypointAttrName) {
+    if (auto moduleOp = dyn_cast<ModuleOp>(op)) {
+      if (!llvm::any_of(moduleOp.getOps(), [](auto &op) {
```

Comment:
We don't need this to be generic. Also, let's not shadow the `op` variable from the parent scope.
```suggestion
      if (!llvm::any_of(moduleOp.getOps(), [](Operation *nestedOp) {
```